### PR TITLE
To Do list: CRUD

### PR DESCRIPTION
**Pull request by HunterX-7**
## In this Project, I:

- Followed GitHub flow.
- Use HTML&CSS and JavaScript Linters.
- Write meaningful commit messages in the most important parts of the process.
- Setup a project with the following features:

  > Implement a function for adding a new task (add a new element to the array).
  > Implement a function for deleting a task (remove an element from the array).
  > Implement a function for editing task descriptions.
  > Add the property index set to the value of the new array length, which updates if changes in the list are made.
  > All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/modules/functionality.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+function addToDo() {
+  const task = addToList.value;
+
+  todos.push({
+    description: task,
+    completed: false,
+    index: todos.length,
+  });
+  addToList.value = '';
```
Comment (rloterh) [inline src/modules/functionality.js]: - [x] _Index_ property of added item does not match new array length. For instance, if a 5th item is added to the list, the index of that item should equal to 5. Kindly ensure to improve upon the same.

**Mismatch in index property and array length:**
image

File: README.md [hunk 2]
```diff
@@ -88,6 +88,16 @@ The following test will run automatically as you try to pull request: -Lighthous
```
Comment (rloterh) [inline README.md:97]: - [x] OPTIONAL
Live demo link on _line 49_ of your ReadMe is broken, kindly endeavor to fix it.

Comment (rloterh) [inline README.md:97]: - [x] OPTIONAL
Kindly include instruction on how to run your app, under the _Getting Started_ section of your ReadMe

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Pancho, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- CRUD feature is functioning as intended

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (HunterX-7) [reply README.md:97]: I will gladly update the status once I get to the part of deploying the page.
In the mean time I have put a message of  "coming soon".

**Review by levy002 — APPROVED** [review]
## Hello @HunterX-7 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (rloterh) [reply README.md:97]: @HunterX-7 
I think you can comment it out at meantime, till you're ready. 

Cheers! 😃

**Pull request merged**